Description: do not make the std:: symbols in webkit public.
             based on Mike Hommey work in the webkit package.
Author: Sune Vuorela <sune@debian.org>

---
 Source/WebKit.pri     |    1 +
 Source/symbols.filter |    5 +++++
 2 files changed, 6 insertions(+)

--- a/Tools/qmake/mkspecs/features/default_post.prf
+++ b/Tools/qmake/mkspecs/features/default_post.prf
@@ -64,6 +64,8 @@
     -Winit-self \
     $$QMAKE_CXXFLAGS
 
+*-g++*:QMAKE_LFLAGS += -Wl,--version-script,$$PWD/symbols.filter
+
 v8: DEFINES += WTF_USE_V8=1
 else {
     contains(JAVASCRIPTCORE_JIT, yes): DEFINES+=ENABLE_JIT=1
--- /dev/null
+++ b/Tools/qmake/mkspecs/features/symbols.filter
@@ -0,0 +1,5 @@
+{
+local:
+_ZSt*;
+_ZNSt*;
+};
